File: C/EX35.cpp
#include <conio.h>
#include <stdlib.h>
#include <stdio.h>
#include <time.h> 
#include <string.h>
#include <math.h>
#include <locale.h>
#include <ctype.h>

int num[20];
int comp,a,b,mult;

int main()
{

printf("insira  20 numeros:\n");
  
  
for(a=0; a<20; a++)
  {
      scanf("%d", &num[a]);
  }

printf("insira  uma constante multiplicativa:\n");
scanf("%d", &mult);
 
 comp=0;
 
for(b=0; b< 20; b++)
  {
      
      num[b] = num[b]*mult;
  }
           
      printf ("Os numeros inseridos e multiplciados pela constante foram:\n");
      
for(comp=0; comp< 20; comp++)
{
	printf("%d \n",num[comp]);
}

}

File: Python/EX35.py
num = []


print('insira  20 numeros:')
  
  
for i in range(0, 20, 1):
    x=int(input())
    num.append(x)

b=int(input('insira  uma constante multiplicativa:'))

comp=0;
 
for i in range(0, 20, 1):
    num[i] = num[i]*b
  
           
print('Os numeros inseridos e multiplciados pela constante foram:')
      
for i in range(0, 20, 1):
	print("",num[i])
